public/index.html

```diff
@@ -8,8 +8,79 @@
 </head>
 <body>
     <div class="wrap">
-        <h1 class="test-headline">Slidy</h1>
-        <p class="test-paragraph">Залиш свій слід у світі подорожей</p>
-    </div>
+        <div class="container">
+          <div class="left-section">
+            <div class="nav">
+              <div class="nav-logo">
+                <svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="0 0 100 100">
```

Comment:
Export SVG as a file and then set on <img /> tag

Reply:
exported as .png